glide.lock

```diff
@@ -0,0 +1,18 @@
+hash: 39a210b656a4711a62930ff3b42a00875f9b7d6f82475ce6cb0362a9fe0f7df3
+updated: 2017-03-08T23:03:14.49057282-05:00
+imports:
+- name: github.com/golang/glog
```

Comment:
tbh, it might make sense to change this to a structured logger.. I just picked the first logging package I found